client/my-sites/stats/stats-list/action-follow.jsx

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
 import config from '@automattic/calypso-config';
 import { Gridicon } from '@automattic/components';
```

Comment:
I couldn't swap icons on line 78 `reader-following` and `reader-follow` because there are no replacements available at the moment.